war/src/main/less/modules/table.less

```
@@ -62,6 +62,7 @@
         vertical-align: middle;
         padding: var(--table-padding) 0 var(--table-padding)
           var(--table-padding);
+        padding-left: 1.6rem;
```

Perhaps

```suggestion
        padding: var(--table-padding) 0 var(--table-padding)
          1.6rem;
```

unless I'm missing something?

Unsure whether this is a correct approach at all though.

---

Yes, @daniel-beck you are correct, my mistake.